Sources/Core/API/Actor.swift

```diff
@@ -96,6 +96,10 @@ public final class Actor: SceneObject, InstanceHashable, ActionPerformer, Plugga
         return pluginManager.add(plugin, for: self, reuseExistingOfSameType: reuseExistingOfSameType)
     }
 
+    public func plugins<P>(ofType type: P.Type) -> [P] where P : Plugin, Actor == P.Object {
```

Comment:
This is a very minor nitpick, but could you type out this method the same was as it's defined in the protocol? I know that Xcode generates method implementations like this, which is strange - but it would just be awesome if it could match the style of the other methods 😇

Reply:
Whoa, I didn't see that, let me fix real quick.